src/features/options/cardBacks.ts

```diff
@@ -55,7 +55,13 @@ export const CARD_BACK = {
   '53': 'CBOnHitEffect',
   '54': 'CBDaganWhite',
   '55': 'CBSonicDoom',
-  '56': 'CBBrandao'
```

Comment:
I wasn't sure if the key value is important or not - i assume it is, and so appended the new FaBrary card back options at the end instead of adding to the middle with the 2 original FaBrary card back options